include/cucumber-cpp/internal/CukeEngine.hpp

```diff
@@ -95,7 +97,8 @@ class CukeEngine {
      */
     virtual std::string snippetText(const std::string & keyword, const std::string & name, const std::string & multilineArgClass) const = 0;
 
-    virtual ~CukeEngine() {}
+    CUCUMBER_CPP_EXPORT CukeEngine();
```

Comment:
Why did you add these? They aren't necessary in similar situations in the Live codebase.